### PR TITLE
Ak form submit

### PR DESCRIPTION
**Pull request by Mbalcorta**
action kit form, redirects to form protest site.

### CODE REVIEW

File: breaktheinternet/index.html [hunk 1]
```diff
@@ -76,33 +76,36 @@ <h3 class="hidden form-error">Hmmm... That didn't work :/</h3>
...
-                          <div id="ak-fieldbox-name">
-                              <input type="text" placeholder="Full Name" name="name" id="id_name" class="ak-userfield-input" style="margin-right:8px"required>
-                          </div>
-                          <div id="ak-fieldbox-email" class="required">
-                              <input type="text" placeholder="Email" name="email" id="id_email" class="ak-userfield-input" required>
-                          </div>
-                          <div id="ak-fieldbox-zip" class="required">
-                              <input type="text" placeholder="Zip Code" name="zip" id="id_zip" class="ak-userfield-input" style="margin-right:8px">
+          <div class="ak-page-container" id="dp_form" style="display:none">
+          <script>console.log("The load_remember_me snippet for Remember Me is disabled by config.")</script>
```
Comment (tibetsprague) [inline breaktheinternet/index.html]: what is this console doing here?

File: breaktheinternet/js/app.js [hunk 2]
```diff
@@ -47,31 +47,36 @@ $(function(){
...
-
-      $form.find('input').removeAttr('disabled');
-      $button.text(buttonText);
+    $('form').submit(function(e){
+      
+      if (loc.indexOf('org=dp') === -1) {
+        e.preventDefault();
+        var $form = $(this);
+        var params = $form.serialize();
+        console.log(params)
```
Comment (tibetsprague) [inline breaktheinternet/js/app.js]: remove console

File: breaktheinternet/js/app.js [hunk 3]
```diff
@@ -47,31 +47,36 @@ $(function(){
...
+        $('.form-error').hide();
+        $form.find('input').attr('disabled', 'disabled');
+        
+        var $button = $form.find('button')
+        var buttonText = $button.text()
+        $button.text('Saving...');
+        
+        $.post($form.attr('action'), params, function(response) {
+          console.log(response)
```
Comment (tibetsprague) [inline breaktheinternet/js/app.js]: remove

Comment (Mbalcorta) [reply breaktheinternet/js/app.js]: when I remove the if statement the page no longer redirects, I get CORS issues and the button stays disabled with the text 'Saving'.

**Comment by tibetsprague** [comment]
@Mbalcorta looks like you have some conflicts to resolve first

File: breaktheinternet/js/app.js (hunk 3 above)
Comment (tibetsprague) [reply breaktheinternet/js/app.js]: oh i meant remove the console.log

Comment (tibetsprague) [reply breaktheinternet/js/app.js]: all the console.logs

**Comment by Mbalcorta** [comment]
Fixed conflicts.

File: breaktheinternet/js/app.js (hunk 3 above)
Comment (Mbalcorta) [reply breaktheinternet/js/app.js]: all removed

**Comment by tibetsprague** [comment]
@Mbalcorta more conflicts!

**Comment by Mbalcorta** [comment]
conflicts are due to cache on src links changing. Once I get the conflicts resolved it needs to be checked as soon as possible else conflicts show up unless.

**Comment by tibetsprague** [comment]
Why dont you just resolve and then merge yourself

**Comment by Mbalcorta** [comment]
Done. Was unsure if you or zeke needed to do the merge on my work.

**Pull request merged**